controller_manager_tests/test/controller_manager_scripts.test

```diff
@@ -8,6 +8,13 @@
       type: controller_manager_tests/EffortTestController
       joints:
       - hiDOF_joint3
+    controller_groups:
+      group1:
+        - my_controller1
+        - my_controller3
+      group2:
+        - my_controller2
+        - my_controller3
```

Comment:
Should there be some kind of failure case here, like a group where two of the controllers try to claim the same resource? Does the new tool fail gracefully under those circumstances?

Reply:
I am trying a failure case. The script is not that smart to verify this error, but it will print errors.

Reply:
The underlying SwitchController service call should return a `False`:

http://docs.ros.org/lunar/api/controller_manager_msgs/html/srv/SwitchController.html

So I think it's mostly just on the script here to detect that and bubble some kind of error message to the user.

Reply:
Now it uses switch_controllers service which will print this
```
[\ERROR] [\/dummy_app] [\1525462144.701348031]: \Could not switch controllers, due to resource conflict
```
The script will also print an error based on the false return value.

Reply:
👍